tools/run_tests/run_xds_tests.py

```diff
@@ -177,65 +184,232 @@ def get_client_stats(num_rpcs, timeout_sec):
             raise Exception('GetClientStats RPC failed')
 
 
-def wait_until_only_given_backends_receive_load(backends, timeout_sec):
+def wait_until_only_given_instances_receive_load(backends,
```

Comment:
It is hard to understand precisely what the method is doing. I suggest to declare two methods without the `allow_failures` flag.

Reply:
Done